.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule "deps/readies"]
 	path = deps/readies
 	url = https://github.com/RedisLabsModules/readies.git
+[submodule "deps/icu"]
+	path = deps/icu
+	url = https://github.com/unicode-org/icu.git
```

Comment:
@LiorKogan @gkorland please review project's license

Reply:
We are good with the license.  The only requirement is that either
(a) this copyright and permission notice appear with all copies of the Data Files or Software, or
(b) this copyright and permission notice appear in associated Documentation.